js/ui/widget/ui.errors.js

```diff
@@ -218,6 +218,46 @@ module.exports = errorUtils(errors.ERROR_MESSAGES, {
     */
     E1052: '{0} should have the "dataSource" option specified',
 
+    /**
+    * @name ErrorsUIWidgets.E1053
+    */
+    E1053: "The 'buttons' option must include an object or a string items only",
+
+    /**
+    * @name ErrorsUIWidgets.E1054
+    */
+    E1054: "Text Editor button's 'location' property can be 'after' or 'before' only",
```

Comment:
I don't think that we need this error. `location` has a default value. Use it whenever an invalid value is passed.